src/kernels/MIOpenKthvalue.cpp

```diff
@@ -195,3 +195,60 @@ extern "C" __global__ void KthvalueFwd(const IN_OUT_TYPE* input,
                              output_tv,
                              indices_tv);
 }
+
+template <typename DTYPE>
+__device__ void kthvalue_bwd(DTYPE* input_grad,
+                             const DTYPE* output_grad,
+                             const uint64_t* indices,
+                             uint64_t dim_size,
+                             uint64_t dim_stride,
+                             tensor_view_t<4> input_grad_tv,
+                             tensor_view_t<5> output_grad_tv,
+                             tensor_view_t<5> indices_tv)
+{
+    /*
+     * input_grad : {N, C, D, H, W}. Select dim: 2(D)
+     * output_grad/indices : {N, C, H, W} or {N, C, 1, H, W} (if keepDim param in miopen.h = True)
```

Comment:
Actually, `{N, C, H, W}` and `{N, C, 1, H, W}` are no different. You can consider both case of `keepDim` value to be `{N, C, H, W}`

Reply:
solved